test/python/compiler/test_transpiler.py

```diff
@@ -1242,6 +1242,19 @@ def test_delay_converts_to_dt(self):
         out = transpile(qc, dt=1e-9)
         self.assertEqual(out.data[0][0].unit, "dt")
 
+    def test_scheduling_backend_v2(self):
+        """Test that scheduling method works with Backendv2."""
+        qc = QuantumCircuit(2)
+        qc.h(0)
+        qc.cx(0, 1)
+        qc.measure_all()
+
+        backend = FakeMumbaiV2()
+        backend.target.dt = 1e-6
```

Comment:
Could remove this if we merge #8000 first I think, though I'm not sure if it gets weird with backporting. No issue having this here anyhow.

Reply:
Yeah we can remove it after #8000 merges. We'll just have to make sure we backport #8000 first then it'll be fine

Reply:
```suggestion
```